TombEngine/Objects/TR2/Entity/tr2_bird_monster.cpp

```diff
@@ -7,25 +7,24 @@
 #include "Game/items.h"
 #include "Game/Lara/lara.h"
 #include "Game/misc.h"
+#include "Math/Math.h"
 #include "Specific/level.h"
-#include "Math/Random.h"
 #include "Specific/setup.h"
 
-using namespace TEN::Math::Random;
-using std::vector;
+using namespace TEN::Math;
 
 namespace TEN::Entities::Creatures::TR2
 {
 	constexpr auto BIRD_MONSTER_ATTACK_DAMAGE = 200;
 	constexpr auto BIRD_MONSTER_SLAM_CRUSH_ATTACK_RANGE = SQUARE(SECTOR(1));
 	constexpr auto BIRD_MONSTER_PUNCH_ATTACK_RANGE		= SQUARE(SECTOR(2));
 
-	#define BIRD_MONSTER_WALK_TURN_RATE_MAX ANGLE(4.0f)
+	const auto BIRD_MONSTER_WALK_TURN_RATE_MAX = ANGLE(4.0f);
```

Comment:
Should be `constexpr`?

Reply:
`ANGLE()` is not a lambda, so this line only works with `const`. Good enough for now.

Reply:
Please apply `constexpr` to all functions in `TombEngine/Math/Legacy.cpp` and then to all `ANGLE()` calls.